src/orion/core/resolve_config.py

```diff
@@ -75,7 +75,7 @@ def is_exe(path):
 # list containing tuples of
 # (environmental variable names, configuration keys, default values)
 ENV_VARS_DB = [
-    ('ORION_DB_NAME', 'name', 'Orion'),
+    ('ORION_DB_NAME', 'name', 'Oríon'),
```

Comment:
I'm not ideologically opposed to the accent in strings and so on, on the contrary (we have "Université de Montréal" at a couple places), but the name of the DB itself would be pushing it IMO.